packages/ui/v2/core/layouts/AppsLayout.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ComponentProps } from "react";
+
+import { VerticalTabItemProps, HorizontalTabs } from "@calcom/ui/v2/core/navigation/tabs";
+
+import Shell from "../Shell";
+
+const tabs: VerticalTabItemProps[] = [
+  {
+    name: "app_store",
+    href: "/apps",
+  },
+  {
+    name: "installed_apps",
+    href: "/apps/installed",
+  },
+];
+
+export default function AppsLayout({
+  children,
+  ...rest
+}: { children: React.ReactNode } & ComponentProps<typeof Shell>) {
+  return (
+    <Shell {...rest}>
+      <div className="flex flex-col p-2 md:p-0 xl:flex-row ">
+        <div className="block xl:hidden">
```

Comment:
Still a draft but thought I'd point out this minor NIT - when xl we still show the sidebar - currently  we get two sets of tabs 

```suggestion
        <div className="block lg:hidden">
```